addons/cubefs/v3.2/docker/fluid_config_init.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright 2022 The Fluid Authors.
```

Comment:
I think the Copyright year should be changed to 2023. thanks.

Reply:
Ok